patches/llvm-all/linux-next

```diff
@@ -0,0 +1 @@
+linux
```

Comment:
We should avoid symlinks as it makes it harder to maintain the patchsets.

While this is convenient, it is possible that one of the patches get picked up before others and we would need to remove this symlink then copy the patch that was not applied to this folder anyways.

Reply:
Removed the symlink and also squashed all the patches to a single commit, hope it's okay.